### PR TITLE
Mention installation of manim sideview extension 

### PR DESCRIPTION
**Pull request by Rickaym**
## Further Information and Comments

I was told that it would be appropriate to insert an appendage for installation of manim-sideview as mentioned in this document (it's an extension for manim that provides an integrated preview and features from other community projects in vscode)!

## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

**Comment by hydrobeam** [comment]
Hmmm, I don't really think we should put an extension for a specific IDE in the docs

**Comment by Darylgolden** [comment]
Contrary to what @naveen521kk thinks, I think this extension should be listed on the plugins site (but that would take some work for the site to accept non PyPi plugins).

**Comment by Rickaym** [comment]
Yeah, I was initially curious about mentioning it in the plugins site but I was instructed to do so here. If it's not appropriate, I can help contribute the github action for pulling plugins to render the vscode marketplace with extensions following the same regex sampling. What do you guys think?

**Comment by naveen521kk** [comment]
> Hmmm, I don't really think we should put an extension for a specific IDE in the docs

Why not? I have seen many projects have a page in their docs about Editor Support for example [Python](https://docs.python.org/3/using/editors.html), [Rust](https://www.rust-lang.org/tools), [tailwind](https://tailwindcss.com/docs/editor-support#intelli-sense-for-vs-code). I guess it should be fine to have a page for editors (probably under the installation section).

> Contrary to what @naveen521kk thinks, I think this extension should be listed on the plugins site (but that would take some work for the site to accept non PyPi plugins).

That would be quite some _work_ to do(probably like kinda starting the site from scratch). 
IMHO, plugins for manim are something that extends Manim features and "Manim Showcase" isn't actually extending features for Manim, so I would still not accept it to be listed in plugin page.

**Comment by behackl** [comment]
First, thanks for your efforts @Rickaym – I think your VS code extension is quite useful, and I support advertising somewhere.

I think that the documentation is a better place than the plugins site -- strictly speaking, the VS Code extension *is not a Manim plugin*, and while we could start to list all sorts of associated software on our plugins site, I feel that it would ultimately cause confusion about the terminology. Plugins should be, as Naveen mentions, Python modules that extend the library in some way.

With that being said, I think that the best place for this is (similar to what exists on this PR already) in a "Editor" section of <https://docs.manim.community/en/stable/installation.html>, or a separate "Tips and Tricks" subpage of the installation section.

**Comment by Rickaym** [comment]
I'm really glad to be of service!

If we could do it, then I think having a brief editor section at the bottom of the `installations.html` similar to the jupyter section will be fitting, and also since there's not much unique content (as for the user guide of the extension) to suffice for a new page.

**Comment by behackl** [comment]
> I'm really glad to be of service!
> 
> If we could do it, then I think having a brief editor section at the bottom of the `installations.html` similar to the jupyter section will be fitting, and also since there's not much unique content (as for the user guide of the extension) to suffice for a new page.

That sounds like a good approach. Once we have more content that should be added there, we can still consider moving it to a separate page. 👍

**Comment by Rickaym** [comment]
Definitely, if in the future we have more content to add, I think having a subpage will be of great addition.

I've amended the commits here, let me know if I should change anything.

File: docs/source/installation.rst [hunk 1]
```diff
@@ -102,6 +102,21 @@ like that yourself:
...
+
+If you're using Visual Studio Code you can install an extension called
+`Manim Sideview` which provides automated rendering and an integrated preview
+of the animation inside the editor.
+
+The extension can be installed through the marketplace or by pressing install `here
+<https://marketplace.visualstudio.com/items?itemName=Rickaym.manim-sideview>`__.
+
+Further instruction and support regarding the extension can be found in its
+`repository <https://github.com/Rickaym/Manim-Sideview>`__ or the marketplace webpage.
```
Comment (behackl) [inline docs/source/installation.rst:118]: Three paragraphs seems a bit much, allow me to make a suggestion for shortening this (while still keeping all relevant information):

```suggestion
of the animation inside the editor. The extension can be installed through the
`marketplace of VS Code <https://marketplace.visualstudio.com/items?itemName=Rickaym.manim-sideview>`__.
```

**Review by behackl — APPROVED** [review]
Left two comments with suggestions, please consider applying them – otherwise this LGTM.

**Pull request merged**